package/MDAnalysis/analysis/contacts.py

```diff
@@ -387,14 +387,18 @@ class Contacts(AnalysisBase):
        :attr:`timeseries` results are now stored in a
        :class:`MDAnalysis.analysis.base.Results` instance.
     """
+    # Error message for wrong selection type:
+    select_error_message = "selection must be either string or a " \
+                           "static AtomGroup"
```

Comment:
MDAnalysis tries to follow PEP8 where possible, as part of it, the recommendation is that line continuation should [use implicit line continuation where possible](https://peps.python.org/pep-0008/#:~:text=The%20preferred%20way%20of%20wrapping,a%20backslash%20for%20line%20continuation).

```suggestion
    select_error_message = ("selection must be either string or a "
                            "static AtomGroup")
```

Reply:
Fixed in commits made since this comment.